client/public/contribute.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Yari",
+    "description": "The next-gen app powering MDN Web Docs.",
+    "repository": {
+        "url": "https://github.com/mdn/yari",
+        "license": "MPL-2.0",
+        "tests": "https://github.com/mdn/yari/actions"
+    },
+    "participate": {
+        "home": "https://wiki.mozilla.org/MDN",
+        "docs": "https://github.com/mdn/yari/blob/master/README.md",
+        "mailing-list": "https://discourse.mozilla.org/c/mdn",
+        "chat": {
+            "url": "https://mozilla.slack.com/archives/CLTBV3W9X"
```

Comment:
Perhaps we should link here instead? https://chat.mozilla.org/#/room/#mdn:mozilla.org